janus.c

```diff
@@ -3551,7 +3552,50 @@ gboolean carbyne_janus_transport_is_sanityhealthcheck_resources_available(janus_
 		JANUS_LOG(LOG_WARN,"SGW not Available\n");
 	}
 
-  	return TRUE;
+
+	//checking if cert is ok
+	FILE *certStatusFile = NULL;
+	char * certStatusLine = NULL;
+	size_t certStatuslen = 0;
+	int certStatusStatus = TRUE;
+	certStatusFile = fopen(CERT_STATUS, "r");
+	if (certStatusFile == NULL)
+	{
+		JANUS_LOG(LOG_ERR,"Failed opening cert status file\n");
+		certStatusStatus = FALSE;
+	}
+	else
+	{
+		if((getline(&certStatusLine, &certStatuslen, certStatusFile)) != -1) 
+		{
+			if(strstr(certStatusLine,"true") == NULL)
+			{
+				JANUS_LOG(LOG_ERR,"Cert will expire or already expired \n");
+				certStatusStatus = FALSE;
+			}
+		}
+		else
+		{
+			JANUS_LOG(LOG_ERR,"no line was found in cert status file\n");
+			certStatusStatus = FALSE;
+		}
+	}
+	// free memeory
+	if (certStatusFile != NULL) 
+	{
+		fclose(certStatusFile);
+	}
+	if (certStatusLine != NULL)
+	{ 
+		free(certStatusLine);
+	}
+
+	// check status at the end
+	if (certStatusStatus != TRUE) 
```

Comment:
you can just return certStatusStatus

Reply:
instead of returning TRUE as well?
if so i will need to also address all other tests in the function